### PR TITLE
popup form + open + close on esc & clickout

### PR DESCRIPTION
**Pull request by benyossef27**
hi everybody!
check out the popup i made for work
(hope i'm doning this right)
shabat shalom

### CODE REVIEW

**Comment by resdenia** [comment]
Hey! Thank you! @benyossef27  nice job!!!
I would add probably some small comments to PR
But looks amazing!

File: work/blocks/popup/__button/popup__button.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.popup__button{
+    align-items: center;
+    padding: 6px 16px;
+    width: 280px;
+    height: 42px;
+    width: 280px;
+    height: 42px;
```
Comment (resdenia) [inline work/blocks/popup/__button/popup__button.css]: In that case you have duplicate of height and width.

**Review by resdenia — CHANGES_REQUESTED** [review]
Please see my comment. I assume it make cleaner your PR

**Review by MariaKonstantinov — APPROVED** [review]

**Review by resdenia — APPROVED** [review]
Nice! From my end I can approve.

**Review by Sashikbear — APPROVED** [review]
Awesome work, please merge

**Review by Omriruvio — APPROVED** [review]
Very nice, good job :)

**Comment by resdenia** [comment]
@benyossef27  Hey, can you please merge your changes! Thank you for your work.

**Pull request merged**